### DISCUSSION TITLE
GT 610 support

### DISCUSSION THREAD

**Discussion by w32u** [chris1111/Legacy-Video-patch#33] [Q&A]
It's said that nvidia 4xx and 5xx are supported, and they are based on Fermi architecture. I have GT 610, which is Fermi too and it works in High Sierra. Will this patch work with it?
Also I have found out that GT 610 is a rebrand of the GT 520 (GF119 gpu, fermi)

**Comment by chris1111** [comment]
GT 610 work good Yosemite and bellow, if you try that card on other macOS the mousse or keyboard freeze
